lib/ofx/parser/ofx220.rb

```diff
@@ -16,9 +16,14 @@ def build_account_info(node)
           description:       node.search('desc').inner_text.upcase,
           bank_id:           nested_account_info.search('bankid').inner_text,
           id:                nested_account_info.search('acctid').inner_text,
-          type:              ACCOUNT_TYPES[node.search('accttype').inner_text.to_s.upcase],
+          type:              fetch_account_type(node),
         })
       end
+
+      def fetch_account_type(node)
```

Comment:
I don't think you need this method here as it's inheriting from `ofx102`

Reply:
Both the methods are different especially the line below
ACCOUNT_TYPES[node.search("bankacctfrom > accttype").inner_text.to_s.upcase]
ACCOUNT_TYPES[node.search('accttype').inner_text.to_s.upcase]